docs/solidity/decryption.md

```diff
@@ -0,0 +1,32 @@
+# Decryption and control structures
+
+## Decryptions
+
+We allow explicit decryption requests for any encrypted type.
+The values are decrypted through the distributed decryption protocol and are stored on-chain.
+
+### Example
```

Comment:
Maybe add example with require() ?

Reply:
I mean another example (one with `return` and one with `require()`)

Reply:
Good idea, done ✅